common/protob/messages-management.proto

```diff
@@ -87,6 +86,7 @@ message Features {
     optional BackupType backup_type = 31;       // type of device backup (BIP-39 / SLIP-39 basic / SLIP-39 advanced)
     optional bool sd_card_present = 32;         // is SD card present
     optional bool sd_protection = 33;           // is SD Protect enabled
+    optional bytes state = 34;                  // state, used mainly to indicate what passphrase was cached
```

Comment:
Shouldn't we just name this as `session`?

Reply:
I'm not against. @matejcik?

Reply:
i agree, let's call it session

Reply:
Ok. So, `session` or `session_id`? I am in favour of the latter